CRM/Group/Page/AJAX.php

```diff
@@ -50,7 +50,7 @@ public static function getGroupList() {
       CRM_Utils_JSON::output($groups);
     }
     else {
-      $params = CRM_Core_Page_AJAX::defaultSortAndPagerParams();
+      $params = array_merge($params, CRM_Core_Page_AJAX::defaultSortAndPagerParams());
```

Comment:
Okay, used validateParams() as I can't see `getSanitizedParams()` is merged into master yet.